src/controllers/confirm.company.controller.ts

```diff
@@ -10,7 +10,7 @@ import { isActiveFeature } from "../utils/feature.flag";
 export const get = async (req: Request, res: Response, next: NextFunction) => {
   try {
     const company: CompanyProfile = await getCompanyProfile(req.query.companyNumber as string);
-    return res.render(Templates.CONFIRM_COMPANY, { company });
+    return res.render(Templates.CONFIRM_COMPANY, { company, templateName: Templates.ERROR_404 });
```

Comment:
shouldnt this be 
return res.render(Templates.CONFIRM_COMPANY, { company, templateName: Templates.CONFIRM_COMPANY });

Reply:
fixed 👍